packages/js/src/generators/library/library.spec.ts

```diff
@@ -1144,357 +1141,6 @@ describe('lib', () => {
           outputs: ['{options.outputPath}'],
         });
       });
-
```

Comment:
We should probably keep some test cases here to assert that the release configuration is correctly set. I understand we are testing everything in the new helper, but having some simpler test cases here validates that the `@nx/js:library` generator is correctly invoking the helper. We don't need to keep all the existing tests, but at least one or two to ensure the helper is correctly invoked and prevent potential regressions.

Reply:
added it back